modules/indexExchangeBidAdapter.js

```diff
@@ -419,6 +426,9 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     _IndexRequestData.targetAggregate = {'open': {}, 'private': {}};
 
     if (!utils.hasValidBidRequest(bidArr[0].params, requiredParams, ADAPTER_NAME)) {
+      for (var i = 0; i < bidArr.length; i++) {
+        passOnBid(bidArr[i].placementCode);
+      }
```

Comment:
@snapwich
Our endpoint had changed in this PR - [#826](https://github.com/prebid/Prebid.js/pull/826#). The site id and slot id information from the first slot were treated uniquely in our original endpoint.  Now that endpoint had changed, there are no needs to reject all bids if the first slot is invalid. Since this validation is an existing behaviour and introduced no behaviour changes to IX adapter, this issue should not block this particular pull request. In the later release, we will remove this first slot validation logic.